dali/pipeline/operator/operator.h

```diff
@@ -173,7 +173,7 @@ class DLL_PUBLIC OperatorBase {
     using namespace std;  // NOLINT
     static_assert(is_arithmetic_or_half<remove_reference_t<T>>::value || is_enum<T>::value,
                   "The eligible diagnostic entry types are arithmetic types or enum");
-    if (!diagnostics_.emplace(move(name), val).second) {
+    if (!diagnostics_.emplace(std::move(name), val).second) {
```

Comment:
What's the warning about?

Reply:
Clang reports usage of unqualified calls to `move`. I think it sees it as a potential threat even if `using namespace std` is used.

Reply:
Fun fact: in C++20 you don't have to use `std::` to call `std::` functions on types from `std::`, because the scope of argument-dependent lookup has been increased - so, you can do:
```C++
auto foo() {
    std::vector<int> v;
    return move(v);   // argument-dependent lookup kicks in
}
```
Same goes for `copy` and basically all stuff from `algorithm` (finally!).

Reply:
`--std=c++20` still returns this warning.